src/data_types.ts

```diff
@@ -25,18 +25,9 @@ export interface AbstractDataType<T> {
  */
 
 export abstract class DataType {
-  dataType: string;
+  dataType: string = '';
```

Comment:
`compilerOptions.strictPropertyInitialization` complains about unset class field

Reply:
which is quite common in our test models, such as 

```ts
class Foo extends Bone {
  @Column()
  id: bigint;
}
```

added a default value to make downstream projects happy for now